src/bluetooth-fw/nimble/nimble_type_conversions.c

```diff
@@ -44,7 +44,41 @@ bool pebble_device_to_nimble_conn_handle(const BTDeviceInternal *device, uint16_
   int rc = ble_gap_conn_find_by_addr(&addr, &desc);
   if (rc == 0) {
     *handle = desc.conn_handle;
+  } else {
+    PBL_LOG_D(LOG_DOMAIN_BT, LOG_LEVEL_ERROR,
+              "failed to find connection handle for addr" BT_DEVICE_ADDRESS_FMT,
+              BT_DEVICE_ADDRESS_XPLODE(device->address));
   }
```

Comment:
handle the exception, not the rule (i.e., invert this `if` conditional)

Reply:
Done.